uaclient/config.py

```diff
@@ -218,6 +217,35 @@ def metering_timer(self, value: int):
         self.cfg["ua_config"]["metering_timer"] = value
         self.write_cfg()
 
+    @property
+    def poll_for_pro_license(self) -> bool:
+        # TODO: when polling is supported
+        #     1. change default here to True
+        #     2. add this field to UA_CONFIGURABLE_KEYS
+        return self.cfg.get("ua_config", {}).get("poll_for_pro_license", False)
+
+    @poll_for_pro_license.setter
+    def poll_for_pro_license(self, value: bool):
+        if "ua_config" not in self.cfg:
+            self.cfg["ua_config"] = {}
+        self.cfg["ua_config"]["poll_for_pro_license"] = value
+        self.write_cfg()
+
+    @property
+    def polling_error_retry_delay(self) -> int:
+        # TODO: when polling is supported
+        #     1. add this field to UA_CONFIGURABLE_KEYS
+        return self.cfg.get("ua_config", {}).get(
+            "polling_error_retry_delay", 600
```

Comment:
Thanks for providing the additional config option here. Though it may very well be above-and-beyond, making it configurable will give folks an option to backoff if UA is hitting frequent errors related to accessing GCP IMDS.